setup.py

```diff
@@ -27,8 +27,8 @@ def read(filename):
 
     packages=find_packages(exclude=('tests',)),
 
-    install_requires=['numpy>=1.19.1', 'matplotlib', 'scipy', 'tqdm', 'tensorflow_probability==0.11.1',
-                      'maxent@git+https://github.com/ur-whitelab/maxent.git', 'pandas', 'geopandas==0.8.1', 'geoplot==0.4.1', 'networkx==2.5'],
+    install_requires=['numpy>=1.19.1', 'matplotlib', 'scipy', 'tqdm', 'tensorflow_probability==0.11.1', 'jupyter',
```

Comment:
Might want to loosen these a bit. Not sure if exactly 0.11.1 TFP is required.

Reply:
removed version